src/odemis/acq/move.py

```diff
@@ -1063,26 +1086,41 @@ def __init__(self, microscope):
             self.postures.append(MILLING)
         if model.MD_FAV_FIB_POS_ACTIVE in stage_md:
             self.postures.append(FIB_IMAGING)
+        # If there is no known fixed sample z for FM, compute it here and store it for later use
+        if not stage_md.get(model.MD_FM_POS_SAMPLE_ACTIVE):
+            # For FM imaging, fix the imaging plane so that all features always lie on the same plane.
+            # Pick a sane default for the sample stage z using grid 1.
+            sem_grid1_pos = dict(stage_md[model.MD_SAMPLE_CENTERS][POSITION_NAMES[GRID_1]])
+            sem_grid1_pos.update(stage_md[model.MD_FAV_SEM_POS_ACTIVE])
+            sem_grid1_pos_fm = self._transformFromSEMToMeteor(sem_grid1_pos)
```

Comment:
```suggestion
            sem_grid1_pos_fm = self._transformFromSEMToMeteor(sem_grid1_pos, fix_fm_plane=False)
```
Mostly in case we restart Odemis GUI and the GRID1 has changed, we want to use this new one.